web/src/redux/apis/endpoints/Transaction.endpoint.ts

```diff
@@ -1,18 +1,21 @@
 import {HTTP_METHOD} from 'constants/Common.constants';
 import {TracetestApiTags} from 'constants/Test.constants';
-import Transaction, {TRawTransaction} from 'models/Transaction.model';
+import Transaction, {TRawTransaction, TRawTransactionResource} from 'models/Transaction.model';
 import TransactionService from 'services/Transaction.service';
 import {TTestApiEndpointBuilder} from 'types/Test.types';
 import {TDraftTransaction} from 'types/Transaction.types';
 
+const defaultHeaders = {'content-type': 'application/json', 'X-Tracetest-Augmented': 'true'};
```

Comment:
Including the augmented header

Reply:
not sure how this works, but just to be safe: for requesting the yaml definition, you need to NOT pass the header.

Reply:
Yep we have that covered 😄